examples/constructor-minimal/README.md

```diff
@@ -0,0 +1,21 @@
+# Constructor Minimal Example
+
+This demo shows how to use `pixi` to generate explicit `conda` environment specs,
+and [`constructor`][constructor] to use these specs to build platform-specific,
+offline-ready installers.
+
+## How to use?
+
+Make sure you have `pixi` available in your terminal.
+
+Navigate to this directory and run:
+
+```shell
+# Build the installer
+pixi run build
+
+# Install the built installer
+pixi run start
```

Comment:
It now has no error when running this, @bollwyvl, but it's unclear to me what I can expect after this? It doesn't install `micromamba` for me. Which I assumed, any tips?